Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -103,7 +103,7 @@
   "loc.messages.dotnetCommandFailed": "Dotnet command failed with non-zero exit code on the following projects : %s",
   "loc.messages.noProjectFilesFound": "Project file(s) matching the specified pattern were not found.",
   "loc.messages.noPublishFolderFoundToZip": "A publish folder could not be found to zip for project file: %s.",
-  "loc.messages.noWebProjctFound": "No web project was found in the repository. Web projects are identified by presence of either a web.config file or wwwroot folder in the directory.",
+  "loc.messages.noWebProjectFound": "No web project was found in the repository. Web projects are identified by presence of either a web.config file or wwwroot folder in the directory. You can uncheck `Publish Web Projects` if your project doesn't follow the aforementioned web project criteria.",
```

Comment:
You can set `Publish Web Projects` property to false (publishWebProjects: false in yml) if your project doesn't following this convention or if you want to publish projects other than web projects.